modules/aruco/tutorials/aruco_detection/aruco_detection.markdown

```diff
@@ -101,55 +102,59 @@ The generated image is:
 
 ![Generated marker](images/marker23.jpg)
 
-A full working example is included in the ```create_marker.cpp``` inside the module samples folder.
+A full working example is included in the `create_marker.cpp` inside the module samples folder.
 
 Note: The samples now take input via commandline via the [OpenCV Commandline Parser](http://docs.opencv.org/trunk/d0/d2e/classcv_1_1CommandLineParser.html#gsc.tab=0). For this file the example parameters will look like
 @code{.cpp}
```

Comment:
> http://docs.opencv.org/trunk/d0/d2e/classcv_1_1CommandLineParser.html#gsc.tab=0

Please replace external links to builtin doxygen autolinks.
Perhaps this should work well:
```
via the OpenCV cv::CommandLineParser
```

Reply:
Done (three occurrences).